packages/fxa-settings/src/models/Account.ts

```diff
@@ -1333,6 +1333,10 @@ export class Account implements AccountData {
         },
       },
     });
+    cache.writeQuery({
```

Comment:
@vbudhram We can look improving this in FXA-7626 or FXA-7184, we have some other similar comments